src/main/java/net/sf/jabref/external/WriteXMPEntryEditorAction.java

```diff
@@ -55,11 +55,16 @@ public void actionPerformed(ActionEvent actionEvent) {
         List<File> files = new ArrayList<>();
 
         // First check the (legacy) "pdf" field:
-        entry.getFieldOptional(FieldName.PDF).ifPresent(pdf -> FileUtil
-                .expandFilename(pdf, panel.getBibDatabaseContext().getFileDirectory("pdf")).ifPresent(files::add));
+        entry.getFieldOptional(FieldName.PDF)
```

Comment:
It does indeed, but that is what Eclipse gives me when reformatting the whole expression...